Bang/GameEvents/CardEffects/CatBalouActionHandler.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.Linq;
+using Bang.GameEvents.CardEffects.States;
+using Bang.Players;
+using Bang.PlayingCards;
+
+namespace Bang.GameEvents.CardEffects
+{
+    internal class CatBalouHandler : CardActionHandler
+    {
+        public override HandlerState ApplyEffect(Game.Gameplay gameplay, Player victim, BangGameCard card)
+        {
+            if (victim.Hand.Count + victim.ActiveCards.Count == 0)
+                return new ErrorState();
+
+            var codeToCard = new Dictionary<ClosedHandCard, BangGameCard>();
+            for (int i = 0; i < victim.Hand.Count; i++)
+            {
+                codeToCard[new ClosedHandCard {Code = i}] = victim.Hand[i];
+            }
+            
+            var response = new ChooseOneCardResponse
```

Comment:
I think there is no difference between random dropping card from hand or giving a choice. But random is easier :)

Reply:
Fixed